requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+-r requirements.txt
```

Comment:
Thanks! I should put this in "software-development" then...

Just wanted to mention that you need to put this at the top of ``requirements.txt`` => ``-r requirements/conda.txt`` for it to make sense :sweat_smile: